### DISCUSSION TITLE
Removing Sections from Homepage

### DISCUSSION THREAD

**Discussion by roheetnarayanan** [wangchucheng/hugo-eureka#116] [Q&A]
This is probably a weird question. But is it possible remove the Experience and Vintage section from the homepage?

I'm aware of controlling this in hugo-academic through the **active** parameter. But not quite sure, how it works in eureka.

Any help here is much appreciated.

**Comment by wangchucheng** [comment]
You can just delete those files (eg. experiences.md) in your project ;-)


________________________________
发件人: Roheet Narayanan ***@***.***>
发送时间: Friday, June 11, 2021 4:33:25 PM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: Subscribed ***@***.***>
主题: [wangchucheng/hugo-eureka] Removing Sections from Homepage (#116)


This is probably a weird question. But is it possible remove the Experience and Vintage section from the homepage?

I'm aware of controlling this in hugo-academic through the active parameter. But not quite sure, how it works in eureka.

Any help here is much appreciated.

―
